src/components/ChatLog.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ( { entries, likeMessage, unlikeMessage } ) => {
+    const chatEntries = entries.map(entry => {
+        return (
+            <ChatEntry
+                id={entry.id}
```

Comment:
We should also use the id for the key prop since we're iterating over many entries to create many ChatEntrys:
```javascript
 <ChatEntry
     id={entry.id}
     key={entry.id}
     // the rest of your props for your component
/>